app/assets/javascripts/discourse/app/components/about-page-user.gjs

```diff
@@ -0,0 +1,56 @@
+import Component from "@glimmer/component";
+import { or } from "truth-helpers";
+import avatar from "discourse/helpers/avatar";
+import { prioritizeNameInUx } from "discourse/lib/settings";
+import { userPath } from "discourse/lib/url";
+import i18n from "discourse-common/helpers/i18n";
+
+export default class AboutPageUser extends Component {
+  get prioritizeName() {
+    return prioritizeNameInUx(this.args.user.name);
+  }
+
+  <template>
+    <div data-username={{@user.username}} class="user-info small">
+      <div class="user-image">
+        <div class="user-image-inner">
+          <a
+            href={{userPath @user.username}}
+            data-user-card={{@user.username}}
+            aria-hidden="true"
+          >
+            {{avatar @user imageSize="large"}}
+          </a>
+        </div>
+      </div>
+      <div class="user-detail">
+        <div class="name-line">
+          <a
+            href={{userPath @user.username}}
+            data-user-card={{@user.username}}
+            aria-label={{i18n
+              "user.profile_possessive"
+              username=@user.username
+            }}
+          >
+            <span class="username">
+              {{#if this.prioritizeName}}
+                {{@user.name}}
+              {{else}}
+                {{@user.username}}
+              {{/if}}
+            </span>
+            <span class="name">
+              {{#if this.prioritizeName}}
+                {{@user.username}}
+              {{else}}
+                {{@user.name}}
+              {{/if}}
+            </span>
+          </a>
+        </div>
+        <div class="title">{{or @user.title ""}}</div>
```

Comment:
The title would render as nothing if it's undefined/null (and I don't think we have to worry about `false` value 😃)

```suggestion
        <div class="title">{{@user.title}}</div>
```